doc/README.md

```diff
@@ -174,23 +178,64 @@ pprof can generate graphical reports on the DOT format, and convert them to
-represented as a node. Solid edges represent a direct connection between
-entries, while dotted edges represent a connection where some intermediate nodes
-have been removed. Nodes are removed using heuristics to limit the size of
+represented as a node. Nodes are removed using heuristics to limit the size of
 the graph, controlled by the *nodecount* option.
 
-The size of each node represents the flat weight of the node, and the width of
-each edge represents the cumulative weight of all samples going through
-it. Nodes are colored according to their cumulative weight, highlighting the
-paths with the highest cum weight.
-
-* **-dot:** Generates a report in .dot format. All other formats are generated from
-  this one.
+* **-dot:** Generates a report in .dot format. All other formats are generated
+  from this one.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+* **Flat Value**: the value of a location itself, is indicated by the box
+  size.
+    *   Nodes with larger box have larger flat values.
+    *   Nodes with smaller box have smaller flat values.
+
+* **Cum Value**: the value of a location plus all of its descendants, is
+  indicated by the nodes' color.
+    *   Redder nodes have greater cum values.
+    *   Greyer nodes have smaller cum values.
+
+* **Dashed Edges**: some locations between the two connected locations were
+  removed.
+
+* **Solid Edges**: one location directly calls the other.
+
+* **Thicker & Redder Edges**: more resources (i.e. larger value) were used
+  along that path.
+
+* **Thinner & Greyer Edges**: less resources (i.e. smaller value) were used
+  along that path.
+
+* **"(inline)" Edge Marker**: the call has been inlined into the caller.
+
+Let's consider the following example graph:
+
+![callgraph](images/callgraph.svg)
+
+* In terms of nodes, we notice that:
+  * `NewWriter` has a small flat value and a small cum value because the
+    node is small and grey
+  * `deflate` has a large flat value and a large cum value because the node
+    is large and red
+  * `Flush` has a small flat value and a large cum value because the node is
+    small and red.
 
+* In terms of edges, we take a look at:
+  * the edge between `(*Writer).Write` and `(*compressor).write`: 
+    * since it is a dashed edge, some nodes were removed between those two
+    * since it is red, more resources were used in call stacks between those two
+      nodes 
+  * and the edge between `NewWriter` and `newobject`:
+    * since it is solid, there are no nodes between those two (i.e. it was a
+      direct call)
+    * since it is grey, fewer resrouces were used in call stacks between those
+      two nodes.
+  
```

Comment:
Users I've talked to tend to think the really thick dashed lines and really thin dashed lines represent different things. It would be nice if this example clarified that both types of dashed lines are the same.

Reply:
Done.